build.sh

```diff
@@ -22,9 +22,23 @@ set -e
 
 MODE="$1"
 
+if ! hash xcpretty 2> /dev/null; then
+  echo "xcpretty not installed. Try gem install xcpretty"
+  trap - EXIT
+  exit 1
+fi
+
+if hash pod 2> /dev/null; then
+  pod install --no-repo-update
```

Comment:
Pod install doesn't update repo by default :)

Reply:
Hmm ... didn't know that! 🧐

Think I'm mixing up `brew` and `pod`

```bash
james@Jamess-iMac: ~
$ grep "pod install" "${HOME}/.bash_eternal_history" | uniq | grep -c "no-repo-update"
339
```

Will remove.

Reply:
Yeah, `pod update` does, `pod install` doesn't. I'm thinking about if we should add `Gemfile` and install cocoapods via bundler, as well as `pandoc` or other stuff you have included here :)

Reply:
> `Gemfile` and install cocoapods via bundler, as well as `pandoc` or other stuff you have included here :)

can you install a binary via a Bundler?